addons/account/demo/account_demo.py

```diff
@@ -95,6 +96,32 @@ def _post_load_demo_data(self, company=False):
             except (UserError, ValidationError):
                 _logger.exception('Error while posting demo data')
 
+        # We want the initial balance to be set in he equity_unaffected account
+        demo_bank_statement_1 = self.ref('demo_bank_statement_1')
+        if demo_bank_statement_1:
+            cid = company.id or self.env.company.id
+            current_year_earnings_account = self.env['account.account'].search([
+                *self.env['account.account']._check_company_domain(cid),
+                ('account_type', '=', 'equity_unaffected')
+            ], limit=1)
+            demo_bank_statement_1.line_ids[0].line_ids[1].write({
+                'account_id': current_year_earnings_account.id,
+            })
```

Comment:
```suggestion
            demo_bank_statement_1.line_ids[0].line_ids[1]['account_id'] = current_year_earnings_account.id
```
Nitpick (same below).